LLVM.Imports.Core.pas

```diff
@@ -1771,56 +2039,153 @@ procedure LLVMSetUnwindDest(InvokeInst: TLLVMValueRef; B: TLLVMBasicBlockRef); c
  * Functions in this group only apply to instructions for which
  * LLVMIsATerminatorInst returns true.
  *
- * @{
- *)
-
-(**
- * Return the number of successors that this terminator has.
  *
- * @see llvm::Instruction::getNumSuccessors
- * )
+ *)
 
 
  (**
  * Return a pointer to the next non-debug instruction in the same basic block as 'this', or nullptr if no such instruction exists.
  *
  * @see llvm::Instruction::getNextNonDebugInstruction // Add by Pigreco
  *)
-function LLVMgetNextNonDebugInstruction(Istruz: TLLVMValueRef): TLLVMValueRef; cdecl; external CLLVMLibrary;
+//function LLVMgetNextNonDebugInstruction(Istruz: TLLVMValueRef): TLLVMValueRef; cdecl; external CLLVMLibrary;
 
 (**
 * Return a pointer to the next non-debug instruction in the same basic block as 'this', or nullptr if no such instruction exists.
 *
 * @see llvm::Instruction::PrevNonDebugInstruction // Add by Pigreco
 *)
-function LLVMgetPrevNonDebugInstruction(Istruz: TLLVMValueRef): TLLVMValueRef; cdecl; external CLLVMLibrary;
+//function LLVMgetPrevNonDebugInstruction(Istruz: TLLVMValueRef): TLLVMValueRef; cdecl; external CLLVMLibrary;
```

Comment:
are these supposed to be commented out?(why not removed then?)

Reply:
fix